pkg/redisnode/node_test.go

```diff
@@ -60,3 +74,32 @@ func TestAdminCommands(t *testing.T) {
 	node.ForgetNode()
 	node.StartFailover()
 }
+
+func TestNode_UpdateNodeConfigFile(t *testing.T) {
+	type fields struct {
+		config     *Config
+		IP         string
+		Addr       string
+		RedisAdmin redis.AdminInterface
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
remove this or fill it  :dancing_men:

Reply:
the function is already tested in another test. so I will remove this test.